File: src/main/java/fr/epyi/metropiamod/item/ModItemGroups.java
package fr.epyi.metropiamod.item;

import net.minecraft.item.ItemGroup;
import net.minecraft.item.ItemStack;

public class ModItemGroups {

    public static final ItemGroup METROPIA_UTILITIES = new ItemGroup("metropiaUtilitiesTab") {
        @Override
        public ItemStack createIcon() {
            return new ItemStack(ModItems.CLOTH.get());
        }
    };

}

File: src/main/java/fr/epyi/metropiamod/item/ModItemGroups.kt
package fr.epyi.metropiamod.item

import net.minecraft.item.ItemGroup
import net.minecraft.item.ItemStack

object ModItemGroups {
    val METROPIA_UTILITIES: ItemGroup = object : ItemGroup("metropiaUtilitiesTab") {
        override fun createIcon(): ItemStack {
            return ItemStack(ModItems.CLOTH.get())
        }
    }
}